tools/build-wheels.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+set -e
```

Comment:
https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail

Reply:
`set -euxo pipefail`

Reply:
Even better:
```bash
if [ -n "$DEBUG" ]
then
  set -x
fi

set -euo pipefail
```

Reply:
Do you mean I should change it to `set -euxo pipefail`?
Sounds good but please left it for next PRs.
I hope you'll add the proposal into your work on travis polishing.
Right now I want to fix travis deployment for getting 2.3 release ASAP.
Every change requires a waiting for travis for completion.

Reply:
sure